test/graphemetest.c

```diff
@@ -49,7 +51,7 @@ int main(int argc, char **argv)
                 else
                     i++;
             }
-            glen = utf8proc_map(utf8, j, &g, UTF8PROC_CHARBOUND);
+            glen = utf8proc_map(utf8, j, &g, UTF8PROC_CHARBOUND | UTF8PROC_OLDENCODE);
```

Comment:
Yes, true... I'll add some further testing, and make sure it gets all of the edge cases that were broken before... there are no API changes though.